backend/internal/system/error/serviceerror/error.go

```diff
@@ -51,6 +55,16 @@ func CustomServiceError(svcError ServiceError, errorDesc string) *ServiceError {
 	return err
 }
 
+// I18nServiceError defines a service error structure with i18n support.
+// This is the new error type that should be used for services being migrated to i18n.
```

Comment:
I hope the expectation here is to remove the existing CustomServiceError, once we migrate all the errors to this format.